modules/mutes.js

```diff
@@ -0,0 +1,47 @@
+let setMutedRole = async (message, args) => {
+  let roleId = args[0]
+  let guild = message.client.guilds.get(process.env.BOT_GUILD)
+  let roleName = guild.roles.get(roleId)
+  let returnMessage
+  if (roleName) {
+    message.client.guildData.get(process.env.BOT_GUILD).mutedRole = roleId
+    returnMessage = `Role ${roleName} has been set as the muted role.`
+  } else {
+    returnMessage = `That role does not exist dummy.`
+  }
+  return message.channel.send(returnMessage)
+}
+
+let mute = async (message, args) => {
+
+}
+
+module.exports = {
+  name: 'mute',
+  commands: [
+    {
+      name: 'setmutedrole',
+      description: 'Sets the role to use as the mute role',
+      examples: ['setmutedrole yeet'],
+      secret: true,
```

Comment:
shouldn't be secret

Reply:
Adding to this since I can't find the code review button: the example should be an id-like number, not a string (which yeet is not a good one at that).